### PR TITLE
Release 29 review fixes

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

File: sru/release-29/test-key-rename-failure.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+echo "###########################################"
+lxc exec $name -- ls /etc/apt/trusted.gpg.d/
+echo -e "###########################################\n"
+
+# Make sure the services are still enabled
+echo -e "\n* Check that services are still enabled"
+echo "###########################################"
+lxc exec $name -- pro status --wait
+echo -e "###########################################\n"
+
```
Comment (renanrodrigo) [inline sru/release-29/test-key-rename-failure.sh:84]: At this point, I think it is useful to run `apt update` and show that no error regarding the signing keys is presented

File: sru/release-29/test-key-rename-failure.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+echo -e "\n* List gpg keys"
+echo "###########################################"
+lxc exec $name -- ls /etc/apt/trusted.gpg.d/
+echo -e "###########################################\n"
+
+# Modify the postinst script to fail after renaming just one key
+lxc exec $name -- sed -i "s/^\s\+SERVICES=.*/error/g" /var/lib/dpkg/info/ubuntu-advantage-tools.postinst
+echo -e "\n* Running postinst script again"
+echo "###########################################"
+lxc exec $name -- /var/lib/dpkg/info/ubuntu-advantage-tools.postinst configure || true
```
Comment (orndorffgrant) [inline sru/release-29/test-key-rename-failure.sh:70]: I think this command is the way you're supposed to do it
```suggestion
lxc exec $name -- dpkg-reconfigure ubuntu-advantage-tools || true
```

File: sru/release-29/test-key-rename-failure.sh [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+lxc exec $name -- pro disable esm-apps
+lxc exec $name -- pro status
+echo -e "###########################################\n"
+
+# Check that the GPG key is still there
+echo -e "\n* List gpg keys"
+echo "###########################################"
+lxc exec $name -- ls /etc/apt/trusted.gpg.d/
+echo -e "###########################################\n"
+
```
Comment (orndorffgrant) [inline sru/release-29/test-key-rename-failure.sh:97]: Maybe add `pro enable esm-apps` again to see the state after that

**Comment by lucasmoura** [comment]
@orndorffgrant @renanrodrigo updated

**Review by basak — APPROVED** [review]
This looks good to me - all my review comments that I asked for SRU purposes are addressed. Thank you!

**Pull request merged**